calcom/cal.com
# feat: add confirmed booking webhook event

## What does this PR do?

This adds a webhook trigger for an opt-in booking getting confirmed.
I'm not 100% sure if this is the correct way to add it, I kind of just figured it out by reading the others. Any changes, please let me know :pray: 

## Type of change


## How should this be tested?

1. Set up a webhook test endpoint
2. set a webhook integration with it's URL
3. select the booking confirmed option
4. save it.
5. create an event that is opt-in and book it
6. then confirm it 
7. you should get a payload sent to the test endpoint.

## Checklist:
